### ISSUE TITLE
How to disable the camelCase warning?

### ISSUE THREAD

**Issue by Smartik89** [WordPress-Coding-Standards/WordPress-Coding-Standards#1155]
Hi.

I declare the methods in camelCase format.
How can I disable the warnings generated by wpcs? 

I've commented this line in WordPress-Core->ruleset.xml, but I'm sure that this is not the best method:
```
<!-- <rule ref="WordPress.NamingConventions.ValidFunctionName"/> -->
```
Please advice.

![screenshot-10](https://user-images.githubusercontent.com/1436534/30750297-be6cda1c-9fbe-11e7-97c6-eb3add619534.jpg)

**Comment by JDGrimes** [comment]
You're right, it is best not to modify WPCS directly. What you can do to tweak the configuration is create your own custom ruleset. Just add a `phpcs.xml` file in the root of your project, and put something like this into it:

```xml
<?xml version="1.0"?>
<ruleset name="Custom">
	<description>My custom ruleset.</description>

	<rule ref="WordPress-Core">
		<exclude name="WordPress.NamingConventions.ValidFunctionName" />
	</rule>
</ruleset>
```

You can replace `WordPress-Core` with `WordPress-Extra` or whatever standard you were using. You may also have to configure your IDE to use your custom ruleset.

**Comment by Smartik89** [comment]
Great. :) 
Thank you.
